src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -59,15 +60,14 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
   }
 
   async checkAudioOutput(
-    audioOutputDeviceInfo: MediaDeviceInfo,
+    audioOutputDeviceInfo: MediaDeviceInfo | string,
     audioOutputVerificationCallback: () => Promise<boolean>,
     audioElement: HTMLAudioElement = null
   ): Promise<CheckAudioOutputFeedback> {
     try {
-      const audioOutputDeviceId =
-        audioOutputDeviceInfo && audioOutputDeviceInfo.deviceId
-          ? audioOutputDeviceInfo.deviceId
-          : '';
+      const audioOutputDeviceId = audioOutputDeviceInfo
+        ? DefaultDeviceController.getIntrinsicDeviceId(audioOutputDeviceInfo)
+        : '';
```

Comment:
I think you want `null` here.

Reply:
The previous value is also '' and I check bindAudioMix, it also use ''.